### ISSUE TITLE
No "Register" button when WebID is disabled

### ISSUE THREAD

**Issue by gklyne** [solid/node-solid-server#1188]
This may fall into the category of "it's a feature, not a bug".  But here goes...

We're using node-solid-server primarily as an LDP server for some research software we're developing, but are using the directory structure created for a single-user Solid deployment;  specifically, the `/public` directory.  Initially, I installed and tested with WebID enabled, performed the initial user registration, had software do the authentication dance, and all was fine.  

But other software doesn't do authentication, so we tried setting up a new server with WebID disabled so we could use the basic LDP capabilities.  Having done this, we found that there was no "Register" button on the server front page, and without registering a user, some of the directory structures (`/public`, etc.) appear not be be created.

Our workaround is to start with WebID enabled, perform the initial user registration, then edit the `config.json` file to disable webID, and this seems to work for us (so far).

This behaviour was slightly surprising for us, and it did take us a little while to figure out what was happening.  If this is considered to be the desired behaviour, I'd suggest that a note in the documentation describing how to use the server as a plain LDP platform might be helpful.

**Comment by jaxoncreed** [comment]
Yeah, I think you hit the nail on the head in terms of it being a feature.

To confirm, what you would want is for the server to generate the same folder structure as it would with the webId enabled? 

Also just interested, what is your use case for using Node Solid Server as an LDP rather than one like Trellis

**Comment by gklyne** [comment]
Our immediate use case is software development/experimentation, where dealing with authentication just adds complications.  And that we've found that node solid server works well for us  as a vanilla LDP server compared with others we've looked at :-o

Longer-term, I think we'll want to use the authentication/authorization too (which is a good reason to work with the generated folder structure.

Our experimental software is written to work with the folder structure created with WebID enabled.  It would be nice to (have an option to) create that structure without enabling WebId.  But that's sufficiently marginal that I think a more reasonable ask would be: 

1. having a brief note in the docs about how to set up to run as a plain LDP server (an option that is mentioned, but when I last looked I couldn't find a description of how to do this), and

2. more informative messages when trying to get started without WebID enabled - this issue arose because we were getting a pretty obscure message about some resource being missing.

This is all pretty low-priority.  For now, we're looking good as I've updated our local notes for getting the solid server running [Create basic structure for user data (LDP container)](https://github.com/oerc-music/meld-cli-tools/blob/master/notes/20190208-solid-server-install-run.md#create-basic-structure-for-user-data-ldp-container).

**Comment by jaxoncreed** [comment]
Yeah, sorry about it being low priority. If you'd like to make a pull request, I'll check it out.
